packages/wallet/store/middleware.test.js

```diff
@@ -10,6 +10,10 @@ import { tokenMap } from '@token/configuration/tokens';
 import * as transactionApi from '@transaction/utilities/api';
 import { getAutoLogInData } from '@common/utilities/login';
 import history from '@common/utilities/history';
+import blockActionTypes from '@block/store/actionTypes';
+import transactionActionTypes from '@transaction/store/actionTypes';
+import walletActionTypes from '@wallet/store/actionTypes';
```

Comment:
Since the wallet action types is in the same folder, it might as well be imported from `./actionTypes`

Reply:
yhea true